tests/automation_framework/tests/work_order_tests/test_getresult.py

```diff
@@ -0,0 +1,257 @@
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "work_order_getresult.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (
+            verify_test(
+                submit_response, 0,
+                self.test_obj.build_request_output['pre_test_output'],
+                self.test_obj.build_request_output['action_obj'])
+            is TestStep.SUCCESS.value)
+
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.work_order_get_result
+    @pytest.mark.test_workordergetresult_workorderid_different
+    @pytest.mark.sdk
+    def test_workordergetresult_workorderid_different(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordergetresult_workorderid_different.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (
+                check_negative_test_responses(
+                    submit_response,
+                    "Invalid work order Id")
+                is TestStep.SUCCESS.value)
+
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.work_order_get_result
+    @pytest.mark.test_workordergetresult_workorderid_specialchar
+    @pytest.mark.sdk
+    def test_workordergetresult_workorderid_specialchar(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "workordergetresult_workorderid_specialchar.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_getresult(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+        logger.info("submit_response: \n%s\n", submit_response)
+
+        assert (
+                check_negative_test_responses(
+                    submit_response,
+                    "Invalid work order Id")
+                is TestStep.SUCCESS.value)
+
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+    @pytest.mark.work_order_get_result
```

Comment:
Change all these markers to our naming convention of one word for the API. Ex: test_workorder_getresult. Same would apply for all tests.

Reply:
done. marker is changed to "workordergetresult" for all the tests.